filebeat/module/logstash/_meta/docs.asciidoc

```diff
@@ -0,0 +1,40 @@
+== logstash module
+
+This module parses logstash regular logs and the slow log, it will support the plain text format
+and the JSON format (--log.format json). The default is the plain text format.
+
+
+[float]
+=== Compatibility
+
+This modules was tested with logs from Logstash 5.6 and 6.0.
+
+[float]
+===  This module comes with a sample dashboard
+
+{INSERT IMAGE AFTER REVIEW}
+
+[float]
+=== Log fileset settings
```

Comment:
This part of the docs is being refactored in https://github.com/elastic/beats/pull/5341. It might make sense to update it already to the new style of module documentation.

Reply:
Thanks for the pointer! I will do that.